src/RegisterPage.cpp

```diff
@@ -107,19 +108,38 @@ RegisterPage::RegisterPage(QWidget *parent)
           tr("A server that allows registration. Since matrix is decentralized, you need to first "
              "find a server you can register on or host your own."));
 
+        QFont font;
+
+        error_username_label_ = new QLabel(this);
+        error_username_label_->setFont(font);
```

Comment:
Do you actually need to set the font here?

Reply:
I set here, because I need to add error_username_label to form_layout

Reply:
No, I mean, doesn't it use the right font anyway?

Reply:
error_label_ = new QLabel(this);
        error_label_->setFont(font);
It's was in error_label, and I used it here  too.

Reply:
I think you don't need it

Reply:
Remove setFont ?

Reply:
Yeah, and the font. It just sets the label to use the default font, which it should be using already anyway.